README.md

```diff
@@ -62,6 +62,7 @@ Tested on:
 | [@ec-europa/ecl-forms-text-inputs](framework/components/ecl-forms/ecl-forms-text-inputs) | [![npm version](https://badge.fury.io/js/%40ec-europa%2Fecl-forms-text-inputs.svg)](https://badge.fury.io/js/%40ec-europa%2Fecl-forms-text-inputs) |
 | [@ec-europa/ecl-forms-textareas](framework/components/ecl-forms/ecl-forms-textareas) | [![npm version](https://badge.fury.io/js/%40ec-europa%2Fecl-forms-textareas.svg)](https://badge.fury.io/js/%40ec-europa%2Fecl-forms-textareas) |
 | [@ec-europa/ecl-galleries](framework/components/ecl-galleries) | [![npm version](https://badge.fury.io/js/%40ec-europa%2Fecl-galleries.svg)](https://badge.fury.io/js/%40ec-europa%2Fecl-galleries) |
+| [@ec-europa/ecl-lang-select-pages](framework/components/ecl-lang-select-pages) | [![npm version](https://badge.fury.io/js/%40ec-europa%2Fecl-lang-select-pages.svg)](https://badge.fury.io/js/%40ec-europa%2Fecl-lang-select-pages) |
```

Comment:
That's why I've added `My component is listed in the root README` in the PR template. Better late than never 😉